staff/albert-escolar/blog-semantics/entries.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
```

Comment:
@Escoly folder name should be blog! please update asap and inform us. Thanks.

Reply:
Name changed!